### PR TITLE
Add autocomplete widget to searchbar

### PR DESCRIPTION
**Pull request by purple-waves**
Deployed version for testing: https://autocompletesearch-dot-step269-2020.ts.r.appspot.com

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -28,7 +28,34 @@ function initMap() {
         streetViewControl: false,
         fullscreenControl: false,
     });
+    // Initialise places autocomplete in search bar
+    const searchbar = document.getElementById("location-search-bar");
+    const searchMarker = new google.maps.Marker({
+        map,
+        anchorPoint: new google.maps.Point(0,-29),
```
Comment (roroross) [inline src/main/webapp/script.js]: what does the (0,-29) point/lead to?

Comment (purple-waves) [reply src/main/webapp/script.js]: Not entirely sure, that's just what was on the example

Comment (purple-waves) [reply src/main/webapp/script.js]: It's to do with positioning the icon image relative to the place

Comment (purple-waves) [reply src/main/webapp/script.js]: I've added a comment explaining it

**Review by roroross — COMMENTED** [review]
lgtm

**Review by roroross — APPROVED** [review]

**Review by Techget — APPROVED** [review]

**Pull request merged**